### PR TITLE
CLN: Use `is` operator for comparing with `None` (Pep8)

### PR DESCRIPTION
**Pull request by proinsias**

### CODE REVIEW

File: pandas/tests/test_base.py [hunk 1]
```diff
@@ -293,15 +293,15 @@ def test_none_comparison(self):
 
                 o[0] = np.nan
 
-                result = o == None
+                result = o is None
```
Comment (jreback) [inline pandas/tests/test_base.py]: these all need to be left alone, these are explicity testing ==

**Comment by proinsias** [comment]
I've reverted the changes to that file.
Btw, is it kosher to add comments like PyCharm's "# noinspection PyComparisonWithNone" to specifically try to avoid newbies like me from thinking there's an error there?

**Comment by jreback** [comment]
@proinsias yes having a comment is ok to prevent linters from finding this is fine

**Comment by jreback** [comment]
ping when updated

**Comment by proinsias** [comment]
I've updated the pull request to avoid modifying the test code, and to add #noinspection comments in the test code.

**Comment by jreback** [comment]
thanks!

**Pull request merged**